pkgs/applications/terminal-emulators/mlterm/default.nix

```diff
@@ -19,25 +87,44 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-gfs5cdwUUwSBWwJJSaxrQGWJvLkI27RMlk5QvDALEDg=";
   };
 
-  nativeBuildInputs = [ pkg-config autoconf wrapGAppsHook ];
+  nativeBuildInputs = [
+    pkg-config
+    autoconf
+  ] ++ addIfHas "mlconfig" toolsChoices [
+    wrapGAppsHook
+  ];
   buildInputs = [
-    libX11
+    gtk
+    vte
+  ] ++ lib.optionals (imagelibChoice == "gdk-pixbuf") [
```

Comment:
Nit: If you use `optionals` that often, just inherit it from lib once and stop the `lib.` spam.

Reply:
> Nit: If you use `optionals` that often, just inherit it from lib once and stop the `lib.` spam.

I don't mind this spam that much personally, I copy paste these lines anyway, and it will save me only 4 letters. Will do it if you insist.

Reply:
I won't insist, it just goes against the DRY principle.